### PR TITLE
People Update NY

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update NY

### CODE REVIEW

File: data/ny/retired/Brian-Miller-ad93da57-8a9a-4dc0-bc2c-2488c0331d16.yml [hunk 1]
```diff
@@ -7,18 +7,13 @@ image: https://assembly.state.ny.us/write/upload/member_files/101/headshot/101.j
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
```
Comment (maxknutsen) [inline data/ny/retired/Brian-Miller-ad93da57-8a9a-4dc0-bc2c-2488c0331d16.yml:12]: @bfossen-ce move to leg.

File: data/ny/retired/Jake-Ashby-b03d69c0-0ea1-4524-86e6-c6b20797a2eb.yml [hunk 2]
```diff
@@ -7,18 +7,13 @@ image: https://assembly.state.ny.us/write/upload/member_files/107/headshot/107.j
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
```
Comment (maxknutsen) [inline data/ny/retired/Jake-Ashby-b03d69c0-0ea1-4524-86e6-c6b20797a2eb.yml:12]: @bfossen-ce move to leg.

File: data/ny/legislature/James-Gaughran-089d12fb-2567-487c-8661-d5a930cb0c16.yml [hunk 3]
```diff
@@ -1,30 +1,33 @@
 id: ocd-person/089d12fb-2567-487c-8661-d5a930cb0c16
-name: James Gaughran
+name: Steven D. Rhoads
 given_name: James
 family_name: Gaughran
```
Comment (maxknutsen) [inline data/ny/legislature/James-Gaughran-089d12fb-2567-487c-8661-d5a930cb0c16.yml:4]: @bfossen-ce this is a weird one. It looks like this file should belong to james gaughran who left office on 01-01-2023. 

Seperately, Steven Rhoads needs his own file created https://ballotpedia.org/Steven_Rhoads

File: data/ny/retired/James-Skoufis-c1f048f2-6e1d-4e1b-b161-d97e2f16c1f8.yml [hunk 4]
```diff
@@ -11,18 +11,13 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
   district: '99'
-- type: upper
+- end_date: '2023-01-01'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
   district: '39'
```
Comment (maxknutsen) [inline data/ny/retired/James-Skoufis-c1f048f2-6e1d-4e1b-b161-d97e2f16c1f8.yml:17]: @bfossen-ce move to leg.

File: data/ny/retired/Mark-Walczyk-615c8d33-9652-4163-87b9-0fc82fc09b8c.yml [hunk 5]
```diff
@@ -7,19 +7,13 @@ image: https://assembly.state.ny.us/write/upload/member_files/116/headshot/116.j
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
```
Comment (maxknutsen) [inline data/ny/retired/Mark-Walczyk-615c8d33-9652-4163-87b9-0fc82fc09b8c.yml:12]: @bfossen-ce move to leg.

File: data/ny/retired/Nathalia-Fernandez-2a38ba4b-176e-479f-8083-2ff50aef693e.yml [hunk 6]
```diff
@@ -7,17 +7,13 @@ image: https://assembly.state.ny.us/write/upload/member_files/080/headshot/080.j
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2023-01-01'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
```
Comment (maxknutsen) [inline data/ny/retired/Nathalia-Fernandez-2a38ba4b-176e-479f-8083-2ff50aef693e.yml:12]: @bfossen-ce move to leg.

File: data/ny/retired/Patrick-Burke-85c8422b-5132-4147-bdfc-f1cdec6d4731.yml [hunk 7]
```diff
@@ -10,13 +10,6 @@ roles:
 - type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
   district: '142'
```
Comment (maxknutsen) [inline data/ny/retired/Patrick-Burke-85c8422b-5132-4147-bdfc-f1cdec6d4731.yml:12]: @bfossen-ce move to leg.

File: data/ny/retired/Patrick-M-Gallivan-531f1b36-a809-496b-beb8-ba3cd0bc0c53.yml [hunk 8]
```diff
@@ -7,9 +7,13 @@ image: https://www.nysenate.gov/sites/default/files/styles/160x160/public/09-15-
 party:
 - name: Republican
 roles:
-- type: upper
+- end_date: '2023-01-01'
+  type: upper
   jurisdiction: ocd-jurisdiction/country:us/state:ny/government
```
Comment (maxknutsen) [inline data/ny/retired/Patrick-M-Gallivan-531f1b36-a809-496b-beb8-ba3cd0bc0c53.yml:12]: @bfossen-ce move to leg.

File: data/ny/retired/Assembly-District--39265ac2-b5e7-49fe-ae45-4164ccbc7aa3.yml [hunk 9]
```diff
@@ -0,0 +1,22 @@
+id: ocd-person/39265ac2-b5e7-49fe-ae45-4164ccbc7aa3
+name: Assembly District 23
+image: https://assembly.state.ny.us/write/upload/member_files/000/headshot/000.jpg
+party:
```
Comment (maxknutsen) [inline data/ny/retired/Assembly-District--39265ac2-b5e7-49fe-ae45-4164ccbc7aa3.yml:4]: @bfossen-ce note sure what's happening here. Assembly district 23 is held by stacey G. Pheffer Amato - https://ballotpedia.org/Stacey_G._Pheffer_Amato

File: data/ny/retired/Assembly-District--a245f8a1-a310-4b5e-baaa-1ec2c060721e.yml [hunk 10]
```diff
@@ -0,0 +1,22 @@
+id: ocd-person/a245f8a1-a310-4b5e-baaa-1ec2c060721e
+name: Assembly District 49
+image: https://assembly.state.ny.us/write/upload/member_files/000/headshot/000.jpg
```
Comment (maxknutsen) [inline data/ny/retired/Assembly-District--a245f8a1-a310-4b5e-baaa-1ec2c060721e.yml:3]: @bfossen-ce same thing here. This district is held by Lester Chang - https://ballotpedia.org/Lester_Chang.

File: data/ny/retired/Joe-Angelino-cc6881df-b386-488d-bbc1-71e71b808306.yml [hunk 11]
```diff
@@ -3,22 +3,18 @@ name: Joe Angelino
 given_name: Joe
 family_name: Angelino
 email: angelinoj@nyassembly.gov
```
Comment (maxknutsen) [inline data/ny/retired/Joe-Angelino-cc6881df-b386-488d-bbc1-71e71b808306.yml:5]: @bfossen-ce move to leg.

File: settings.yml [hunk 12]
```diff
@@ -41,6 +41,11 @@ ma:
   - chamber: lower
     district: 11th Plymouth
     vacant_until: 2023-01-14
+ny:
+  vacancies:
```
Comment (maxknutsen) [inline settings.yml:45]: @bfossen-ce lester chang is in lower-49 so it shouldn't be represented as vacant.

**Review by maxknutsen — COMMENTED** [review]
@bfossen-ce good to go once comments are fixed, thanks!

File: data/ny/retired/Assembly-District--39265ac2-b5e7-49fe-ae45-4164ccbc7aa3.yml (hunk 9 above)
Comment (bfossen-ce) [reply data/ny/retired/Assembly-District--39265ac2-b5e7-49fe-ae45-4164ccbc7aa3.yml:4]: At the time of the scrape this seat was still in a recount. Will delete this record

File: data/ny/retired/Assembly-District--a245f8a1-a310-4b5e-baaa-1ec2c060721e.yml (hunk 10 above)
Comment (bfossen-ce) [reply data/ny/retired/Assembly-District--a245f8a1-a310-4b5e-baaa-1ec2c060721e.yml:3]: At the time of the scrape this seat was still in a recount. Will delete this record

**Review by maxknutsen — APPROVED** [review]

**Pull request merged**